cli/lib/kontena/machine/aws/node_provisioner.rb

```diff
@@ -96,21 +98,18 @@ def run!(opts)
 
         ##
         # @param [String] grid
-        # @return [Aws::EC2::SecurityGroup]
+        # @return [Array] Security group id in array
```

Comment:
The instance creation takes an array of group ids and we do not use the actual group info at all after creating it so I thought it would be more clean that way. :)